app/src/test/java/org/oppia/app/testing/StoryTextSizeFragmentTest.kt

```diff
@@ -0,0 +1,80 @@
+package org.oppia.app.testing
+
+import android.content.Intent
+import android.view.View
+import android.widget.SeekBar
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.GeneralClickAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Tap
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.options.STORY_TEXT_SIZE
+import org.oppia.app.options.StoryTextSizeActivity
+import org.oppia.app.utility.OrientationChangeAction.Companion.orientationLandscape
+
+@RunWith(AndroidJUnit4::class)
```

Comment:
Shift this file to a separate `options` package instead of `testing`

Reply:
Done, moved to `testing/options`